### DISCUSSION TITLE
BITGET LISTING PROPOSAL

### DISCUSSION THREAD

**Discussion by Joel020731** [akash-network/community#371] [Ideas]
Greetings - I hope this message finds you well.

My name is Joel and I am here on behalf of Bitget, the world's foremost copy trading platform with a remarkable daily trading volume of $10 billion. I extend a proposition for the inclusion of your token on Bitget. Our platform has forged alliances with global football luminary Lionel Messi and the esteemed football club, Juventus, among other noteworthy achievements.

BitGet offers an all-encompassing trading experience including spot, futures, and copy trading. By enlisting your project on BitGet, you will tap into a vast, engaged community, while we commit to promoting your endeavor through meticulously crafted articles and announcements across our extensive and diverse networks.

This is your chance to ascend to new pinnacles, and we, at Bitget, are eager to assist you in this endeavor. I look forward to a positive response from you so we may discuss deeper about the intricacies of featuring your remarkable project on our platform.
![Rundown on the Bitget platform.pdf](https://github.com/akash-network/community/files/13243897/Rundown.on.the.Bitget.platform.pdf)

These are requirements for listing.

Project name: 

Project abbreviation: “$USD for example”

Project description: 

Project region: 

Is the token/coin launched: 

Website link:

Twitter link:

Telegram link:

Discord link:

Coinmarketcap/coingecko link: 

Other Cex listings: 

Connected with team:

Listing budget:

**Comment by ghost** [comment]
Hey there, thanks for open a discussion on this

Feel free to list AKT token at your discretion as is 100% permissionless  token
